### PR TITLE
Store preferred span name column width in localStorage

### PR DESCRIPTION
**Pull request by bobrik**
Different developers tend to look at different kinds of traces, meaning that they often prefer some set width of span name column in trace view.

Without this patch, the width is set at 25% and it resets on refresh. With this patch the width is remembered in localStorage and preserved on refresh, making it less necessary to adjust.

Posting it without a backing issue since it's a tiny change.

### CODE REVIEW

**Review by yurishkuro — APPROVED** [review]
thanks

File: packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export function newInitialState(): TTraceTimeline {
     detailStates: new Map(),
     hoverIndentGuideIds: new Set(),
     shouldScrollToFirstUiFindMatch: false,
-    spanNameColumnWidth: 0.25,
+    spanNameColumnWidth: parseFloat(localStorage.getItem('spanNameColumnWidth') || '0.25'),
```
Comment (yurishkuro) [inline packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.tsx:55]: prettier linter seems to be failing on this

Comment (bobrik) [reply packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.tsx:55]: I think it's failing below on a missing semicolon. I amended the commit.

Comment (bobrik) [reply packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.tsx:55]: It is now unhappy about `scripts/get-changelog.js`, which I haven't touched, and I'm not sure what to do with that.

Comment (yurishkuro) [reply packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/duck.tsx:55]: I pushed a fix to it https://github.com/jaegertracing/jaeger-ui/commit/ec17f7896d669b21c3b34c9c9fd6946becd9687b, rebasing

**Comment by bobrik** [comment]
Thanks!

**Pull request merged**